### PR TITLE
add msging function

### PR DESCRIPTION
**Pull request by Thumm1993**

### CODE REVIEW

File: hhhh [hunk 1]
```diff
@@ -0,0 +1 @@
+hhhhhhh
```
Comment (Thumm1993) [inline hhhh:1]: please chage datatype

Comment (Thumm1993) [reply hhhh:1]: please take this ASap

**Comment by prashanth357** [comment]
I approve the request

**Review by prashanth357 — COMMENTED** [review]
Simple commit

**Review by Thumm1993 — COMMENTED** [review]
simplw

**Review by prashanth357 — APPROVED** [review]

**Pull request merged**